docker/1.4.0/py2/Dockerfile.cpu

```diff
@@ -0,0 +1,89 @@
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon AI"
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
+
+ARG PYTHON_VERSION=2.7
+ARG PYTORCH_VERSION=1.4.0
+ARG TORCHVISION_VERSION=0.5.0
+ARG MMS_VERSION=1.0.8
+
+# See http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
+ENV LD_LIBRARY_PATH /opt/conda/lib/:$LD_LIBRARY_PATH
+ENV PATH /opt/conda/bin:$PATH
+ENV SAGEMAKER_SERVING_MODULE sagemaker_pytorch_serving_container.serving:main
+ENV TEMP=/home/model-server/tmp
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    cmake \
+    curl \
+    git \
+    jq \
+    libgl1-mesa-glx \
+    libglib2.0-0 \
+    libsm6 \
+    libxext6 \
+    libxrender-dev \
+    openjdk-8-jdk-headless \
+    vim \
+    wget \
+    zlib1g-dev
+
+
+RUN curl -o ~/miniconda.sh -O  https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+ && chmod +x ~/miniconda.sh \
+ && ~/miniconda.sh -b -p /opt/conda \
+ && rm ~/miniconda.sh \
+ && /opt/conda/bin/conda update conda \
+ && /opt/conda/bin/conda install -y \
+    python=$PYTHON_VERSION \
+    cython==0.29.12 \
+    ipython==5.8.0 \
+    mkl-include==2019.4 \
+    mkl==2019.4 \
+    numpy==1.16.4 \
+    scipy==1.2.1 \
+    typing==3.7.4 \
+ && /opt/conda/bin/conda clean -ya
+
+RUN conda install -c \
+    conda-forge \
+    awscli \
+    opencv==4.0.1 \
+ && conda install -y \
+    scikit-learn==0.20.3 \
+    pandas==0.24.2 \
+    pillow==6.2.1 \
+    h5py==2.9.0 \
+    requests==2.22.0 \
+ && conda install \
```

Comment:
Packages from line 38~63 seem to be out of date. Is updating any of these packages causing broken requirements? 

I took a cursory look and found these packages to be outdated. 
cython - 0.29.12->0.29.14
ipython - 5.8.0 -> 7.12.0
mkl-include - 2019.4 -> 2020
mkl - 2019.4 -> 2020
numpy - 1.16.4 -> 1.18.1

Reply:
This applies to all four dockerfiles.